src/app/views/public/landing/landing.controller.coffee

```diff
@@ -0,0 +1,28 @@
+angular.module 'mnoEnterpriseAngular'
+  .controller('LandingCtrl',
+    ($scope, $rootScope, $state, $stateParams, $window, MnoeConfig, MnoeMarketplace, URI) ->
+
+      vm = @
+      vm.isLoading = true
+      $rootScope.publicPage = true
```

Comment:
Shouldn't you do this on the router using `onEnter`?
https://github.com/angular-ui/ui-router/wiki#onenter-and-onexit-callbacks